tests/generate.py

```diff
@@ -816,6 +816,25 @@ def test_eth_manual_addresses_dhcp(self):
 [DHCP]
 UseMTU=true
 RouteMetric=100
+'''})
+
+   def test_eth_use_dns(self):
```

Comment:
This doesn't seem to be properly indented:

```
$ make check-coverage 
...
/usr/bin/pyflakes3 netplan/  tests/test_terminal.py tests/generate.py tests/cli.py tests/test_configmanager.py tests/integration.py
tests/generate.py:821:31: unindent does not match any outer indentation level
   def test_eth_use_dns(self):
                              ^
Makefile:43: recipe for target 'linting' failed
make[1]: *** [linting] Error 1
make[1]: Leaving directory '/home/dja/dev/nplan/netplan'
Makefile:49: recipe for target 'pre-coverage' failed
make: *** [pre-coverage] Error 2
```

If I fix that, I get an enormous amount of test failures because as I noted above you are not setting this to true by default.

Reply:
I can't add a comment pointing to it directly, but you want to set this to true in `handle_network_type` at around line 1482.